video/v1.2/video.jsonld

```diff
@@ -0,0 +1,527 @@
+{
+  "@context": "https://w3id.org/xapi/profiles/context",
+  "id": "https://w3id.org/xapi/video",
+  "type": "Profile",
+  "conformsTo": "https://w3id.org/xapi/profiles#1.0",
+  "prefLabel": {
+    "en": "Video Profile"
+  },
+  "definition": {
+    "en": "The video profile of the xAPI was created to identify and standardize the common types of interactions that can be tracked in any video player."
+  },
+  "seeAlso": "https://github.com/liveaspankaj/xapi-video-cop",
+  "author": {
+    "type": "Organization",
+    "name": "Advanced Distributed Learning(ADL) Initiative"
+  },
+  "versions": [
+    {
+      "id": "https://w3id.org/xapi/video/v1.2",
+      "generatedAtTime": "2018-02-05T10:45:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/video/v1.1"]
```

Comment:
This version needs to be included in the `versions` array as well

Reply:
Thanks @fugu13 !